crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -189,9 +189,11 @@ impl GlobalTransform {
         (self.0.matrix3 * extents).length()
     }
 
-    /// Transforms the given `point`, applying shear, scale, rotation and translation.
+    /// Transforms the given `point` from local space to global space, applying shear, scale, rotation and translation.
     ///
-    /// This moves `point` into the local space of this [`GlobalTransform`].
+    /// To transform a `point` from global to local, use `global_transform.affine().inverse().transform_point(point)`.
+    /// To transform a `vector` from local to global, use `global_transform.affine().transform_vector(vector)`.
+    /// To transform a `vector` global to local, use `global_transform.affine().inverse().transform_vector(vector)`.
```

Comment:
I'd suggest using a code example (that is executed like a test so it must compile).

```suggestion
    /// It can be used like this:
    /// 
    /// ```
    /// # let global_transform = GlobalTransform::default();
    /// # let global_point = Vec3::ZERO;
    /// let local_point = global_transform.affine().inverse().transform_point3(global_point);
    /// # let local_vector = Vec3::ZERO;
    /// let global_vector = global_transform.affine().transform_vector3(local_vector);
    /// # let global_vector = Vec3::ZERO;
    /// let local_vector = global_transform.affine().inverse().transform_vector3(global_vector);
    /// ```
```

This also highlights that the methods are called `transform_vector3` and `transform_point3` :)

Reply:
Thanks for the suggestion. I wrote some doctests which are not the same ones that you wrote, please take a look.